src/Akka.Streams.Kafka.Tests/Integration/CommittableSourceIntegrationTests.cs

```diff
@@ -61,21 +29,22 @@ public async Task CommitableSource_consumes_messages_from_Producer_without_commi
             int elementsCount = 100;
             var topic1 = CreateTopic(1);
             var group1 = CreateGroup(1);
+            var topicPartition1 = new TopicPartition(topic1, 0);
 
-            await GivenInitializedTopic(topic1);
+            await GivenInitializedTopic(topicPartition1);
```

Comment:
See my notes about changing number of partitions and using concrete partition instead of common topic name.
There are lots of changes like that, so I will comment about this only once.

Reply:
Also, I realize that I could use separate test fixture for different tests (that allow 1 or require multiple partitions) - but this will not allow having single container from `docker-compose.yml` to be used for all tests, so I have updated tests to be ready for multiple partitions existance.